Automattic/wp-calypso
# Jetpack Complete+Business provide unlimited premium themes

#### Changes proposed in this Pull Request

* Jetpack Complete and Jetpack Business provide unlimited premium themes
* On a Jetpack site, if you are viewing the theme info page for a premium theme and you do not have access, the Upgrade button will link you to Jetpack Complete instead of Realtime Security
  * See p1639518247313000/1639516498.305800-slack-C0113HTN28N

#### Testing instructions

* Be on the list of special users that can see the softlaunched themes (ask me or @sixhours)
* Create a test JP site (with say, JN)
  * While creating the site, use the extra options to use Jetpack Beta
  * Change the JN jetpack site to point to your sandbox (see: p9dueE-3Ih-p2#comment-6965 )
  * On sandbox, `sudo global-http enable`
* Link to WP.com
* Check out this branch
* Disable isomorphic attribute
```diff
diff --git a/client/sections.js b/client/sections.js
index 628832adfd..b1cceaf469 100644
--- a/client/sections.js
+++ b/client/sections.js
@@ -210,7 +210,7 @@ const sections = [
                module: 'calypso/my-sites/themes',
                enableLoggedOut: true,
                group: 'sites',
-               isomorphic: true,
+               // isomorphic: true,
                title: 'Themes',
        },
        {
@@ -219,7 +219,7 @@ const sections = [
                module: 'calypso/my-sites/theme',
                enableLoggedOut: true,
                group: 'sites',
-               isomorphic: true,
+               // isomorphic: true,
                title: 'Themes',
                trackLoadPerformance: true,
        },
```
* Run calypso locally
* Change to JP site
* Visit the theme showcase with `?flags=themes/premium`
* You should be able to **see** premium themes
* You should not be able to **use** premium themes
* If you try to purchase or activate a premium theme, it should send you to buy JP complete
* After buying JP complete, you should be able to use the premium theme

Related to #59156
